5.37.0/upgradeLegacyRteToCmsFeatureFlag.ts

```diff
@@ -0,0 +1,38 @@
+import { Files } from "../utils";
+import { Project } from "ts-morph";
+import { Context } from "../types";
+import { isFeatureFlagExist } from "../utils/isFeatureFlagExist";
+import { addFeatureFlag } from "../utils/addFeatureFlag";
+
+interface Params {
+    files: Files;
+    project: Project;
+    context: Context;
+}
+
+export const upgradeLegacyRteToCmsFeatureFlag = async (params: Params) => {
+    const { project, files, context } = params;
+
+    context.log.info(
+        "Adding feature flag for legacy Rich Text editor support in the Headless CMS app."
```

Comment:
Let's add the name of the flag, so: "Adding %s feature flag". Do we have a link for this btw? Can we add it here so that users can learn more about tthis?

Reply:
Yes, this article is in our upgrade guide. THis is the link: https://webiny.link/hcms-legacy-rte-support